datastore/additional_data/sources/find_that_charity.py

```diff
@@ -186,10 +187,15 @@ def import_from_path(self, path, org_type=None, replace=False):
             raise self.OrgTypeNotKnownError
 
         if "http" in path:
+            http_headers = {"User-Agent": "360Giving Datastore"}
+            if settings.FTC_HEADER:
```

Comment:
Think it would be good open source citizenship and for dev instances purposes to add an info/warning message that alerts people to not having the postcode information without this setting available and to open an issue with https://github.com/kanedata/find-that-charity if it is needed or something along those lines